### PR TITLE
fix mysql for xenial

### PR DESCRIPTION
**Pull request by eugene-travis**
https://travisci.assembla.com/spaces/Enterprise/tickets/19

### CODE REVIEW

File: lib/travis/build/appliances/services.rb [hunk 1]
```diff
@@ -68,6 +68,8 @@ def apply_mysql
             sh.cmd 'sudo service mysql start', assert: false, echo: true, timing: true
           end
           sh.elif '"$TRAVIS_INIT" == systemd' do
+            sh.cmd "DATADIR=$(sudo mysqld --verbose --help | grep ^datadir | awk '{print $2}')", assert: false, echo: false, timing: false
+            sh.cmd 'sudo find ${DATADIR} -type f -exec touch {} \;', assert: false, echo: false, timing: false
```
[the remark this answers is not in the document]
Comment (eugene-travis) [reply lib/travis/build/appliances/services.rb]: honestly, I do not know. That is a cherry-pick commit from enterprise-2.2 branch

[the remark this answers is not in the document]
Comment (JaroslawSokol) [reply lib/travis/build/appliances/services.rb]: Solution was taken from: https://github.com/docker/for-linux/issues/72
Please double check it.

[the remark this answers is not in the document]
Comment (JaroslawSokol) [reply lib/travis/build/appliances/services.rb]: agree

[the remark this answers is not in the document]
Comment (JaroslawSokol) [reply lib/travis/build/appliances/services.rb]: Mysql works ok on releases from master.
Change is not required.
See comment in ticket  
https://travisci.assembla.com/spaces/Enterprise/tickets/19

**Pull request closed without merging**